specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2019-10-27-preview/openShiftManagedClusters.json

```diff
@@ -0,0 +1,830 @@
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody",
+          "description": "Details about the error."
+        }
+      },
+      "description": "An error response from the Container service."
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "description": "A list of additional details about the error."
+        }
+      },
+      "description": "An error response from the Container service."
+    },
+    "PurchasePlan": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The plan ID."
+        },
+        "product": {
+          "type": "string",
+          "description": "Specifies the product of the image from the marketplace. This is the same value as Offer under the imageReference element."
+        },
+        "promotionCode": {
+          "type": "string",
+          "description": "The promotion code."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "The plan ID."
+        }
+      },
+      "description": "Used for establishing the purchase context of any 3rd Party artifact through MarketPlace."
+    },
+    "OpenShiftRouterProfile": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the router profile."
+        },
+        "publicSubdomain": {
+          "type": "string",
+          "readOnly": true,
+          "description": "DNS subdomain for OpenShift router."
+        },
+        "fqdn": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Auto-allocated FQDN for the OpenShift router."
+        }
+      },
+      "description": "Represents an OpenShift router"
+    },
+    "NetworkProfile": {
+      "properties": {
+        "vnetCidr": {
+          "type": "string",
+          "description": "CIDR for the OpenShift Vnet.",
+          "default": "10.0.0.0/8"
+        },
+        "managementSubnetCIDR": {
+          "type": "string",
+          "description": "CIDR of subnet used to create PLS needed to management the cluster by the ARO Team"
```

Comment:
`...PLS needed to manage the cluster...` ?

Reply:
this is coming from your original PR
I've asked in the team, it seems accurate. Can you suggest a better description?
`CIDR of subnet used to create PLS needed for management of the cluster` - would this work?

Reply:
Sure ! Even better :)